lib/elastic_apm/spies/mongo.rb

```diff
@@ -98,7 +98,7 @@ def pop_event(event)
           span = events.delete(event.operation_id)
           return unless (curr = ElasticAPM.current_span)
 
-          curr == span && ElasticAPM.end_span
+          curr == span && ElasticAPM.end_span(span)
```

Comment:
Will this change actually make a difference? 
Line 101 will only call `end_span` if the span is the last span in the list of `current_spans` (because `curr` is compared to `span`). So passing a span in to `ElasticAPM.end_span` won't change anything.
So I think in order to actually make a difference, we need to get rid of line 99 and just have `ElasticAPM.end_span(span)`, in case it's not the last span in the list of current spans.

Reply:
```suggestion
```